shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +146,146 @@ shared_library("create_flutter_framework_dylib") {
+    "$ocmock_path/OCMock/OCMBlockArgCaller.h",
+    "$ocmock_path/OCMock/OCMBlockArgCaller.m",
+    "$ocmock_path/OCMock/OCMBlockCaller.h",
+    "$ocmock_path/OCMock/OCMBlockCaller.m",
+    "$ocmock_path/OCMock/OCMBoxedReturnValueProvider.h",
+    "$ocmock_path/OCMock/OCMBoxedReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMConstraint.h",
+    "$ocmock_path/OCMock/OCMConstraint.m",
+    "$ocmock_path/OCMock/OCMExceptionReturnValueProvider.h",
+    "$ocmock_path/OCMock/OCMExceptionReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMExpectationRecorder.h",
+    "$ocmock_path/OCMock/OCMExpectationRecorder.m",
+    "$ocmock_path/OCMock/OCMFunctions.h",
+    "$ocmock_path/OCMock/OCMFunctions.m",
+    "$ocmock_path/OCMock/OCMFunctionsPrivate.h",
+    "$ocmock_path/OCMock/OCMIndirectReturnValueProvider.h",
+    "$ocmock_path/OCMock/OCMIndirectReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMInvocationExpectation.h",
+    "$ocmock_path/OCMock/OCMInvocationExpectation.m",
+    "$ocmock_path/OCMock/OCMInvocationMatcher.h",
+    "$ocmock_path/OCMock/OCMInvocationMatcher.m",
+    "$ocmock_path/OCMock/OCMInvocationStub.h",
+    "$ocmock_path/OCMock/OCMInvocationStub.m",
+    "$ocmock_path/OCMock/OCMLocation.h",
+    "$ocmock_path/OCMock/OCMLocation.m",
+    "$ocmock_path/OCMock/OCMMacroState.h",
+    "$ocmock_path/OCMock/OCMMacroState.m",
+    "$ocmock_path/OCMock/OCMNotificationPoster.h",
+    "$ocmock_path/OCMock/OCMNotificationPoster.m",
+    "$ocmock_path/OCMock/OCMObserverRecorder.h",
+    "$ocmock_path/OCMock/OCMObserverRecorder.m",
+    "$ocmock_path/OCMock/OCMPassByRefSetter.h",
+    "$ocmock_path/OCMock/OCMPassByRefSetter.m",
+    "$ocmock_path/OCMock/OCMRealObjectForwarder.h",
+    "$ocmock_path/OCMock/OCMRealObjectForwarder.m",
+    "$ocmock_path/OCMock/OCMRecorder.h",
+    "$ocmock_path/OCMock/OCMRecorder.m",
+    "$ocmock_path/OCMock/OCMReturnValueProvider.h",
+    "$ocmock_path/OCMock/OCMReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMStubRecorder.h",
+    "$ocmock_path/OCMock/OCMStubRecorder.m",
+    "$ocmock_path/OCMock/OCMVerifier.h",
+    "$ocmock_path/OCMock/OCMVerifier.m",
+    "$ocmock_path/OCMock/OCMock.h",
+    "$ocmock_path/OCMock/OCMockObject.h",
+    "$ocmock_path/OCMock/OCMockObject.m",
+    "$ocmock_path/OCMock/OCObserverMockObject.h",
+    "$ocmock_path/OCMock/OCObserverMockObject.m",
+    "$ocmock_path/OCMock/OCPartialMockObject.h",
+    "$ocmock_path/OCMock/OCPartialMockObject.m",
+    "$ocmock_path/OCMock/OCProtocolMockObject.h",
+    "$ocmock_path/OCMock/OCProtocolMockObject.m",
+  ]
+  include_dirs = [ "$ocmock_path" ]
+}
+
+ios_test_flutter_path = rebase_path("$root_out_dir/libios_test_flutter.dylib")
+platform_frameworks_path = "$ios_sdk_path/../../Library/Frameworks/"
+
+# NOTE: This currently only support simulator targets because of the install_name.
+# TODO: Switch the install_name and make the test runner copy the dynamic library
+#       into the testing bundle.
+shared_library("ios_test_flutter") {
```

Comment:
It might be a good idea now to track decommissioning the unused [`FlutterTests`](https://github.com/flutter/engine/blob/c69f21a3493c0277926d2b153ecc407c421521d9/shell/platform/darwin/ios/BUILD.gn#L260) target if this is the way to go to write tests against engine internals with embedding code.

Reply:
That could be useful in the future if someone wanted to do the work of moving the whole build system over to GN.  This addition doesn't necessarily obsolesce target.  I'll add a note.  I'm afraid to rip out something I don't have to touch.